### ISSUE TITLE
calculate the power

### ISSUE THREAD

**Issue by shubhamgoel9** [Py-Droid/Python-Thunder#78]
Before jumping into Pr first comment for assign.All Problem from Edabit. Link is mandatory to add

**Problem statement:**
**Problem Link: ** 

---
Before submitting a PR please Check some details.Check Edabit for problem task

- [ ] fileName formet should be in camelcase
    example- returnTheNextNumber.py
- [ ]  Add label for Problem 
- [ ]  In file there should be problem statement and link to probelm
    example - 
```python
'''
    Probem Task : Power Calculation
    Problem Link : https://edabit.com/challenge/v5gc8FQkDEepkqpfp
'''
*/
def main():
    return a + b
```

This repository is vaild for HacktoberFest2020

Read before making PR - [How to contribute on Github](https://www.dataschool.io/how-to-contribute-on-github/)
Good Luck,

**Comment by Harsh-Udai** [comment]
hey @shubhamgoel9 , i would like to work on this, can you assign this to me.

**Comment by rajat1401** [comment]
Hi @shubhamgoel9 I would like to work on this. please assign it to me.

**Comment by VIPverma01** [comment]
hi @shubhamgoel9  can you assign this yo me please.

**Comment by somkuwaryash** [comment]
Hi @shubhamgoel9 , I would like to work on this.Please assign this to me

**Comment by shubhamgoel9** [comment]
assign me

**Comment by shubhamgoel9** [comment]
solved the issue in PR #81 .Please review it.

**Comment by RamyaThirumalaisamy** [comment]
@aayushi-droid Please assign this task to me,Thanks!
